docs/recipes/CodeSplitting.md

```diff
@@ -0,0 +1,138 @@
+# Code Splitting
+In large web applications, it is often desirable to split up the app code into multiple JS bundles that can be loaded on-demand. This strategy, called 'code splitting', helps to increase performance of your application by reducing the size of the initial JS payload that must be fetched.
+
+To code split with Redux, we want to be able to dynamically add reducers to the store. The default usage of Redux mandates that a single root reducer should to be passed to the `configureStore` call, which makes dynamically adding new reducers more difficult. Below, we discuss some approaches to solving this problem and reference two libraries that provide this functionality.
```

Comment:
Rephrase this a bit.  It's not that "default usage mandates a single root reducer".  It's that there _is_ only one reducer, and the standard usage is to split that up into smaller functions for maintainability.